pkg/cluster/data.go

```diff
@@ -0,0 +1,155 @@
+package cluster
```

Comment:
Does this file duplicate `pkg/basecluster/data.go`? If so, it should be refactored into something common that is used by both packages.

Reply:
Deleted this file and used the pkg/basecluster/data.go